### DISCUSSION TITLE
force login

### DISCUSSION THREAD

**Discussion by KainanSu** [requarks/wiki#5720] [Help / Questions] [answered]
Hi! 
We attempt to use wikijs as our private knowledge base.
Is there any way to force user login before reading pages?

**Comment by Phil-hostwiki** [comment]
You can make Wiki.js inaccessible to non-logged in users.

**Comment by KainanSu** [comment]
@Phil-hostwiki 
Hi! 
But where the setting in wikijs for this method? I really can't find it.

**Comment by KainanSu** [comment, marked as answer]
@Phil-hostwiki 
Oh, I see!

**Set the permissions of group `Guest`.**

I didn't realize that unregistered visitors were designated as Guest by default.

Thanks!
